djangocms_versioning/cms_config.py

```diff
@@ -43,6 +43,17 @@ def handle_admin_classes(self, cms_config):
             [versionable.content_model for versionable in cms_config.versioning],
         )
 
+    def handle_version_admin(self, cms_config):
```

Comment:
I don't think you have any cms config tests for this. We should really be checking that when a versionable is defined in cms_config.py it is appropriately registered in the admin.